### PR TITLE
Add StopNew2013Workflows param to Set-SPOTenant

### PR DESCRIPTION
**Pull request by brianvan2**

### CODE REVIEW

**Comment by trent-green** [comment]
@get-itips can you merge this one as well?

**Comment by get-itips** [comment]
> @get-itips can you review and merge this one as well?

I am sorry, I am afraid I do not have merge permissions. But I'll review.

**Review by get-itips — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1812,6 +1813,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -StopNew2013Workflows
+Prevents creation of new SharePoint 2013 classic workflows.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1817]: I would rephrase:

"Prevents creation of new SharePoint 2013 workflows utilizing the Workflow Manager service."

This will be more familiar to those who leveraged 2013 WFs in an on-prem environment. This is a suggestion, not a requirement -- approved as-is.

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks
